src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -2580,8 +2581,9 @@ class VariantSampleMatrix(val hc: HailContext, val metadata: VSMMetadata,
       genotypeSignature = newEntryType)
   }
 
-  def pcaResults(k: Int = 10, computeLoadings: Boolean = false, computeEigenvalues: Boolean = false, asArrays: Boolean = false): (KeyTable, Option[KeyTable], Option[IndexedSeq[Double]]) = {
-    require(wasSplit)
+  def pca(expr: String, k: Int = 10, computeLoadings: Boolean = false, asArrays: Boolean = false): (IndexedSeq[Double], KeyTable, Option[KeyTable]) = {
+    if (expr == None)
```

Comment:
This expression will always be false.  You probably want `expr == null`.

Reply:
I think I want to take this out altogether--it looks like a holdover from when expr==None meant use the normalized genotype matrix. The check should be done on the python end now.